CMakeLists.txt

```
@@ -139,6 +140,12 @@ if(BUILD_ANIMVIEW)
   message("Building AnimView")
   add_subdirectory(AnimView)
 endif()
+
+if(BUILD_TOOLS)
+	message("Building Tools")
+	add_subdirectory("tools")
```

```suggestion
  message("Building Tools")
  add_subdirectory("tools")
```
Replaced TAB by 2 sapces